cilantro_audit/widgets/completed_audits_list_page.kv

```diff
@@ -10,82 +10,107 @@
-
-            ScrollView:
-                id: scrollable_list
 
                 GridLayout:
-                    id: audit_list
+                    id: headers
                     cols: 5
                     spacing: 10
-                    size_hint_y: None
+                    size_hint_y: 0.1
+
+                    Button:
+                        id: title_header
+                        text: 'Title'
+                        spacing: 10
+                        on_press: root.manager.get_screen(COMPLETED_AUDITS_LIST_PAGE).sort_by_title()
+
+                    Button:
+                        id: date_header
+                        text: 'Date'
+                        spacing: 10
+                        on_press: root.manager.get_screen(COMPLETED_AUDITS_LIST_PAGE).sort_by_date()
+
+                    Button:
+                        id: auditor_header
+                        text: 'Auditor'
+                        spacing: 10
+                        on_press: root.manager.get_screen(COMPLETED_AUDITS_LIST_PAGE).sort_by_auditor()
+
+                    Button:
+                        id: auditor_header
+                        text: 'Severity'
+                        spacing: 10
+                        on_press: root.manager.get_screen(COMPLETED_AUDITS_LIST_PAGE).sort_by_severity()
+
+                    Button:
+                        id: unresolved_header
+                        text: 'Unresolved'
+                        spacing: 10
+                        on_press: root.manager.get_screen(COMPLETED_AUDITS_LIST_PAGE).sort_by_unresolved()
+
+                ScrollView:
+                    id: scrollable_list
 
                     GridLayout:
-                        id: title_col
-                        cols: 1
+                        id: audit_list
+                        cols: 5
+                        spacing: 10
+                        size_hint_y: None
 
-                    GridLayout:
-                        id: date_col
-                        cols: 1
+                        GridLayout:
+                            id: title_col
+                            cols: 1
 
-                    GridLayout:
-                        id: auditor_col
-                        cols: 1
+                        GridLayout:
+                            id: date_col
+                            cols: 1
 
-                    GridLayout:
-                        id: severity_col
-                        cols: 1
+                        GridLayout:
+                            id: auditor_col
+                            cols: 1
 
-                    GridLayout:
-                        id: unresolved_col
-                        cols: 1
+                        GridLayout:
+                            id: severity_col
+                            cols: 1
+
+                        GridLayout:
+                            id: unresolved_col
+                            cols: 1
+
+            GridLayout:
```

Comment:
The same is true for the auditor version.

Reply:
I saw this too, but I was under the impression that this would all be fixed when Bader sweeps through and does his `CilantroButton` conversion. I've been focused more on functionality, rather than UI. Is that not the case?

Reply:
> this would all be fixed when Bader sweeps through and does his CilantroButton conversion.
That is definitely what the plan is. I just wanted to note that so we don't lose track of it easily. The functionality of this is totally good to be in master.